apps/kyb-app/src/components/atoms/Stepper/components/atoms/Breadcrumbs/Breadcrumbs.Label.tsx

```diff
@@ -1,19 +1,15 @@
-import { BaseLabel } from '@/components/atoms/Stepper/components/atoms/BaseLabel/BaseLabel';
 import { BreadcrumbsLabelProps } from '@/components/atoms/Stepper/components/atoms/Breadcrumbs/types';
-import { StepperLabelsMap } from '@/components/atoms/Stepper/types';
-
-export const baseLabelsMap: StepperLabelsMap = {
-  idle: ({ text }) => <BaseLabel variant="idle" text={text} />,
-  warning: ({ text }) => <BaseLabel variant="warning" text={text} />,
-  completed: ({ text }) => <BaseLabel variant="completed" text={text} />,
-};
+import { ctw } from '@ballerine/ui';
 
 export const Label = ({ active, text, state }: BreadcrumbsLabelProps) => {
```

Comment:
Using a `span` for a component called `Label` will cause wrong use. If its in use in forms and other places which expect semantic `label` tag we've hurt native benefits and accessiibility.

Reply:
This label has nothing to do about forms. Its for breadcrumbs.